src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
+@import "./header.scss";
+@import "./card.scss";
+@import "./catalog.scss";
+@import "../utils/variables.scss";
```

Comment:
Consider importing 'variables.scss' before other SCSS files to ensure variables are available for use in other stylesheets.